### PR TITLE
Implemented example for rbx and bulma-pro

### PR DESCRIPTION
**Pull request by Restoration**
Added example with rbx and bulma-pro package

### CODE REVIEW

**Comment by lfades** [comment]
Hi @Restoration , thank you for the PR, I took a look and I like it, there are still some changes required:

- Update the readme to focus only in the example, use the readme of other examples as a guide
- Fix lint issues in the code using `yarn lint-fix` or `npm run lint-fix`
- Remove the static folder, we don't need the favicon in this example
- Remove the `.gitignore` file

File: examples/with-rbx-bulma-pro/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "create-next-example-app",
```
Comment (lfades) [inline examples/with-rbx-bulma-pro/package.json]:
```suggestion
  "name": "with-rbx-bulma-pro",
```

File: examples/with-rbx-bulma-pro/pages/about.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <Layout>
+    <Section>
+      <Title as='h2'>About</Title>
+      <p>That's exactly what we do with Next.js. Instead of PHP, we build the app with JavaScript and React. Here are some other cool features Next.js brings to the table:</p>
+      <ul>
+        <li>Server-rendered by default</li>
+        <li>Automatic code splitting for faster page loads</li>
+        <li>Simple client-side routing (page based)</li>
+        <li>Webpack-based dev environment which supports Hot Module Replacement (HMR)</li>
+        <li>Able to implement with Express or any other Node.js HTTP server</li>
```
Comment (lfades) [inline examples/with-rbx-bulma-pro/pages/about.js]:
```suggestion
```
Remove the line entirely, we don't want this to be one of the main reasons to use Next.js.

Comment (lfades) [inline examples/with-rbx-bulma-pro/pages/about.js]:
```suggestion
```

File: examples/with-rbx-bulma-pro/pages/about.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import {Section, Title} from 'rbx';
+import Layout from '../components/Layout';
+
+const AboutPage = () => (
+  <Layout>
+    <Section>
+      <Title as='h2'>About</Title>
+      <p>That's exactly what we do with Next.js. Instead of PHP, we build the app with JavaScript and React. Here are some other cool features Next.js brings to the table:</p>
```
Comment (lfades) [inline examples/with-rbx-bulma-pro/pages/about.js]: Why does it start like that?, makes no sense to me.

File: examples/with-rbx-bulma-pro/pages/about.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+      <ul>
+        <li>Server-rendered by default</li>
+        <li>Automatic code splitting for faster page loads</li>
+        <li>Simple client-side routing (page based)</li>
```
Comment (lfades) [inline examples/with-rbx-bulma-pro/pages/about.js]:
```suggestion
        <li>Supports static and dynamic routing</li>
```

File: examples/with-rbx-bulma-pro/pages/contact.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import {Section, Title, Field, Label, Control, Input, Textarea, Button} from 'rbx';
+import Layout from '../components/Layout'
+
+const ContactPage = () => (
+  <Layout>
+    <Section>
+      <Title as='h2'>Contact</Title>
```
Comment (lfades) [inline examples/with-rbx-bulma-pro/pages/contact.js]:
```suggestion
      <Title as='h2'>Contact Form Example</Title>
```

File: package.json [hunk 6]
```diff
@@ -107,5 +107,8 @@
   },
   "engines": {
     "node": ">= 8.0.0"
+  },
+  "dependencies": {
```
Comment (lfades) [inline package.json]: Remove this please 🙏

**Review by lfades — CHANGES_REQUESTED** [review]
Update with the changes requested above

File: examples/with-rbx-bulma-pro/pages/about.js (hunk 3 above)
Comment (Restoration) [reply examples/with-rbx-bulma-pro/pages/about.js]: I copied text from [getting start guide](https://nextjs.org/learn/basics/getting-started)

**Review by lfades — APPROVED** [review]

**Comment by simakwm** [comment]
There is a typo in README.
`npm create-next-app --example with-rbx-bulma-pro with-rbx-bulma-pro-app`
It should be
`npx create-next-app --example with-rbx-bulma-pro with-rbx-bulma-pro-app`

**Comment by lfades** [comment]
@simakwm Created https://github.com/zeit/next.js/pull/9494 to fix it, next time you can also create a PR and make the change 💯

**Comment by simakwm** [comment]
Yeah, sorry =)

**Pull request merged**